### PR TITLE
registration-component created , redirect and validation have not been implemented 

### PR DESCRIPTION
**Pull request by lape15**
image

### CODE REVIEW

File: src/Pages/Auth/Register.jsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import logo from "../../assets/budget-it.png";
+import { currency } from "./currency";
```
Comment (Nkemjiks) [inline src/Pages/Auth/Register.jsx:4]: Spacing under. Remember for readability

File: src/Pages/Auth/Register.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    });
+  };
+  const handleRegSubmit = (e) => {
+    e.preventDefault();
+  };
+  return (
+    <div className="auth">
+      <div className="side">
+        <img src={logo} alt="BudgetIt logo" />
+        <h3> Enter your details</h3>
```
Comment (Nkemjiks) [inline src/Pages/Auth/Register.jsx]:
```suggestion
        <h3>Your free account is only 2 seconds away</h3>
```

File: src/Pages/Auth/Register.jsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        <span> Budget your spending.</span>
```
Comment (Nkemjiks) [inline src/Pages/Auth/Register.jsx]:
```suggestion
        <span><em>Budget your spending....</em></span>
```

File: src/Pages/Auth/Register.jsx [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        <span>Budgeting never felt so good.</span>
```
Comment (Nkemjiks) [inline src/Pages/Auth/Register.jsx]:
```suggestion
        <span><em>Budgeting never felt so good....</em></span>
```

Comment (Nkemjiks) [reply src/Pages/Auth/Register.jsx]: Can you also animate them to display like as if you are typing?

File: src/Pages/Auth/Register.jsx (hunk 3 above)
Comment (Nkemjiks) [reply src/Pages/Auth/Register.jsx]: The animation should include this too

File: src/Pages/Auth/Register.jsx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+      </div>
+      <div className="reg-form">
+        <form onSubmit={handleRegSubmit}>
+          <label className="name">First name</label>
```
Comment (Nkemjiks) [inline src/Pages/Auth/Register.jsx]: Wrap each of the input/label in a div. But the the first and last name input, wrap them in another div and use flex box to put the space between them. Using absolute positioning won't be good for some screens

File: src/Pages/Auth/Register.jsx [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+          <select
+            value={user.currency}
+            name="currency"
+            onChange={handleChange}
+            onBlur={handleChange}
+          >
+            {currency.map((cur, index) => {
+              return (
+                <option value={cur.cc} key={index}>
+                  {cur.name} {cur.symbol}
```
Comment (Nkemjiks) [inline src/Pages/Auth/Register.jsx]: Remove the symbol or do this
```suggestion
                 {cur.symbol} - {cur.name} 
```

File: src/styles/_register.scss [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+  }
+}
+
+.auth {
+  display: flex;
+  flex-flow: row wrap;
+  padding: 10px 15px;
+  background: url("../assets/curved.jpeg") no-repeat;
+  background-size: cover;
+  background-position: center center;
```
Comment (Nkemjiks) [inline src/styles/_register.scss:29]:
```suggestion
  background-position: center center;
  height: 100vh;
```

Comment (Nkemjiks) [reply src/styles/_register.scss:29]: The div was not covering the full page on my laptop

File: src/styles/_register.scss [hunk 8]
```diff
@@ -0,0 +1,214 @@
...
+
+  .side {
+    flex-basis: 50%;
+    padding-left: 55px;
+    display: block;
+    height: 520px;
+    padding-top: 20px;
+    color: #fff;
```
Comment (Nkemjiks) [inline src/styles/_register.scss:39]: Add this margin to bring is done a little
```suggestion
    color: #fff;
    margin-top: 100px;
```

File: src/styles/_register.scss [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+      @include mobile() {
+        width: 200px;
+      }
+    }
+    h3 {
+      font-size: 2.7rem;
+      padding: 5px;
+      margin-left: 20px;
+      font-family: "Roboto", sans-serif;
+      letter-spacing: 1px;
```
Comment (Nkemjiks) [inline src/styles/_register.scss:55]:
```suggestion
      letter-spacing: 1px;
      font-weight: 300;
      width: 500px;
```

File: src/styles/_register.scss [hunk 10]
```diff
@@ -0,0 +1,214 @@
...
+      @include mobile() {
+        font-size: 2rem;
+        margin-left: 5px;
+      }
+    }
```
Comment (Nkemjiks) [inline src/styles/_register.scss:67]:
```suggestion
    }
    
```

File: src/styles/_register.scss [hunk 11]
```diff
@@ -0,0 +1,214 @@
...
+    padding-top: 20px;
+    color: #fff;
+    img {
+      width: 150px;
+      margin-left: 20px;
+      margin-top: 40px;
+      @include mobile() {
+        width: 200px;
+      }
+    }
```
Comment (Nkemjiks) [inline src/styles/_register.scss:48]:
```suggestion
    }
    
```

File: src/styles/_register.scss [hunk 12]
```diff
@@ -0,0 +1,214 @@
...
+        letter-spacing: 1px;
+
+        &.name {
+          position: absolute;
+          top: -2px;
+          left: 63px;
+          @include firefox-only() {
+            top: -2px;
+          }
+        }
```
Comment (Nkemjiks) [inline src/styles/_register.scss:154]:
```suggestion
        }
        
```

File: src/styles/_register.scss [hunk 13]
```diff
@@ -0,0 +1,214 @@
...
+        background: #f6f6f6;
+        padding: 5px;
+        margin-right: 8px;
+        border-radius: 5px;
+        margin-bottom: 15px;
+        height: 20px;
+        outline: none;
+        transition: border 0.6s;
+        font-family: "Roboto", sans-serif;
+      }
```
Comment (Nkemjiks) [inline src/styles/_register.scss]:
```suggestion
      }
      
```

File: src/styles/_register.scss [hunk 14]
```diff
@@ -0,0 +1,214 @@
...
+        background: #f6f6f6;
+        padding: 5px;
+        border-radius: 5px;
+        margin-bottom: 15px;
+        height: 30px;
+        margin-top: 23px;
+        outline: none;
+        transition: border 0.6s;
+        font-family: "Roboto", sans-serif;
+      }
```
Comment (Nkemjiks) [inline src/styles/_register.scss]:
```suggestion
      }
      
```

File: src/styles/_register.scss [hunk 15]
```diff
@@ -0,0 +1,214 @@
...
+        margin-bottom: 15px;
+        margin-top: 23px;
+        outline: none;
+        transition: border 0.6s;
+        border: none;
+        letter-spacing: 1px;
+        font-family: "Roboto", sans-serif;
+        border: 1px solid #f9b500;
+        cursor: pointer;
+      }
```
Comment (Nkemjiks) [inline src/styles/_register.scss]:
```suggestion
      }
      
```

File: src/styles/_register.scss [hunk 16]
```diff
@@ -0,0 +1,214 @@
...
+      div.back {
+        padding: 10px;
+        letter-spacing: 1px;
+        font-family: "Roboto", sans-serif;
+        .link {
+          color: #f9b500;
+        }
+      }
+    }
+  }
```
Comment (Nkemjiks) [inline src/styles/_register.scss:327]:
```suggestion
  }
  
```

**Review by Nkemjiks — CHANGES_REQUESTED** [review]
The page is coming nicely but it has some issues. i will add screenshots in the comments

**Comment by Nkemjiks** [comment]
Screenshot 2020-07-04 at 8 55 34 AM

File: src/Pages/Auth/Register.jsx (hunk 4 above)
Comment (lape15) [reply src/Pages/Auth/Register.jsx]: Ohk

File: src/Pages/Auth/Register.jsx (hunk 5 above)
Comment (lape15) [reply src/Pages/Auth/Register.jsx]: Alright

**Pull request merged**